packages/blade/src/components/Card/CardInteractive.stories.tsx

```diff
@@ -0,0 +1,709 @@
+                    </Text>
+                    <Text>
+                      For teams that are scaling up and need advanced features like payment failure.
+                    </Text>
+                  </Box>
+                </CardBody>
+              </Card>
+              <Card
+                as="label"
+                accessibilityLabel="Premium Tier"
+                scaleOnHover
+                isSelected={selected === 'premium'}
+                height="100%"
+              >
+                <CardBody>
+                  <HiddenInput
+                    onChange={(value) => setSelected(value)}
+                    value="premium"
+                    name="pricing-card"
+                  />
+                  <Amount marginBottom="spacing.1" value={20} currency="USD" size="heading-large-bold" />
+                  <Box paddingX="spacing.2">
+                    <Text marginBottom="spacing.3" size="large" type="subtle">
+                      Premium
+                    </Text>
+                    <Text>
+                      Best suited for businesses that need a dedicated account manager and 24x7 support.
+                    </Text>
+                  </Box>
+                </CardBody>
+              </Card>
+            </Box>
+          );
+        };
+
+        export default App;
+        `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/Card/Interactive',
+  component: Card,
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<CardProps>;
+
+const CardTemplate = (): React.ReactElement => {
+  const [selected, setSelected] = React.useState(false);
+
+  return (
+    <Card
+      onHover={() => {
+        console.log('Hovered');
+      }}
+      scaleOnHover
+      width={{ s: '100%', m: '400px' }}
+      isSelected={selected}
+      onClick={() => setSelected(!selected)}
+      accessibilityLabel="Payment Pages Card"
+    >
+      <CardHeader>
+        <CardHeaderLeading
+          title="Payment Pages"
+          subtitle="Card Header Subtitle"
+          prefix={<CardHeaderIcon icon={RupeeIcon} />}
+          suffix={<CardHeaderCounter value={12} />}
+        />
+        <CardHeaderTrailing visual={<CardHeaderBadge variant="positive">NEW</CardHeaderBadge>} />
+      </CardHeader>
+      <CardBody>
+        <Text>
+          Share payment link via an email, SMS, messenger, chatbot etc. and get paid immediately.
+          Accepting payments from customers is now just a link away.
+        </Text>
+      </CardBody>
+      <CardFooter>
+        <CardFooterLeading title="Footer" subtitle="Footer Subtitle" />
+        <CardFooterTrailing
+          actions={{
+            primary: { text: 'Primary', onClick: () => console.log('Primary') },
+            secondary: { text: 'Secondary', onClick: () => console.log('Secondary') },
+          }}
+        />
+      </CardFooter>
+    </Card>
+  );
+};
+
+export const Default = CardTemplate.bind({});
+
```

Comment:
also, the prop table looks bizarre right now it shows styled props as well together. can we separate those props out like we do for other components' stories?

Reply:
Updated.